RELEASE.md

```diff
@@ -14,29 +14,31 @@
 Steps to release a new version of the Jaeger Operator:
 
 
-1. Change the `versions.txt `so that it lists the target version of the Jaeger (if it is required). Don't touch the operator version it will be changed automatically in the next step.
+1. Change the `versions.txt `so that it lists the target version of the Jaeger (if it is required). **Don't touch the operator version**: it will be changed automatically in the next step.
 
-1. Run `OPERATOR_VERSION=1.30.0 make prepare-release`, using the operator version that will be released.
+2. Run `OPERATOR_VERSION=1.30.0 make prepare-release`, using the operator version that will be released.
 
-1. Prepare a changelog since last release.
+3. Run the E2E tests in OpenShift as described in [the CONTRIBUTING.md](CONTRIBUTING.md#an-external-cluster-like-openshift) file. The tests will be executed automatically in Kubernetes by the GitHub Actions CI later.
 
-1. Commit the changes and create a pull request:
+4. Prepare a changelog since last release.
 
-   ```
+5. Commit the changes and create a pull request:
```

Comment:
I think the intention of have 1.  in each bullet is to don't have to re-enumerate if we add more :)

Reply:
Do you want to recover that or is OK with the current approach?

Anyway, if you do this:
```markdown
1.
5.
3.
```

Markdown is rendered as:

1.
5.
3.

But is more legible to follow the steps if you are just reading the MD file

Reply:
ha hhaha unexpected to me, ok then it is ok.